halyard-config/src/main/java/com/netflix/spinnaker/halyard/model/v1/Updateable.java

```diff
@@ -100,4 +105,39 @@
 
     return errors;
   }
+
+  /**
+   * Validate this entire class
+   *
+   * @return The list of errors when validation fails ([] when there are no failures and validation has passed).
+   * @throws IllegalAccessException
+   * @throws NoSuchFieldException
+   */
+  default public List<String> validate() throws IllegalAccessException, NoSuchFieldException {
+    List<String> errors = new ArrayList<>();
+
+    if (this instanceof Account) {
+      Account account = (Account) this;
+      Class aClass = this.getClass();
+
+      errors.addAll(Arrays.stream(aClass.getDeclaredAnnotations())
+          .filter(c -> c instanceof ValidateAccount)                             // Find all ValidateAccount annotations
+          .map(v -> (ValidateAccount) v)
+          .map(ValidateAccount::validators)                                      // Pick of the validators
+          .flatMap(Stream::of)                                                   // Flatten the stream of lists
+          .map(v -> {
+            try {
+              return v.getConstructor(this.getClass()).newInstance(this);        // Construct the validators
+            } catch (Exception e) {
+              throw new RuntimeException(e);
+            }
+          }).filter(v -> !v.skip())                                              // Ignore skippable validators
```

Comment:
This is how the intellij settings shipping with gradle are formatting it, and I'd rather rely on that w/o too much thought about formatting by hand.